src/creature_states.c

```diff
@@ -4183,14 +4183,22 @@ TbBool creature_is_hostile_towards(const struct Thing *fightng, const struct Thi
     return false;
 }
 
-TbBool creature_can_be_hostile(const struct Thing *fightng, const struct Thing *enmtng)
+/* Determines if two creatures are hostile towards each other.
```

Comment:
The point of this comment is you can hover it and it says what it does. So, best if it adds value over the function name. Like 'Determines if two creatures will fight each other even when allied'.

Reply:
That works automatically.

Reply:
![image](https://github.com/user-attachments/assets/a3540ac5-3be6-49d2-bdc1-dd65164f1049)